contracts/test/ProofUtil.sol

```diff
@@ -0,0 +1,20 @@
+// SPDX-License-Identifier: UNLICENSED
+pragma solidity ^0.8.20;
+
+import {Cids} from "../src/Cids.sol";
+import {MerkleVerify, Hashes} from "../src/Proofs.sol";
+
+// Methods for committing to data and generating proofs.
+// These are only used in tests (which verify proofs).
+// These functions provide a spec for the operations which providers should perform off-chain.
+library ProofUtil {
```

Comment:
Were you planning to move more stuff here?  If not I don't understand extracting this piece out and leaving the others, I think at least buildProof and buildTree belong here.

Reply:
The move was strictly based on generateLeaves being shared between multiple test files.

I do see value in moving some other small helpers out here too. `buildProofs` (plural) is tricker to home. It's the spec for how proofs should be built off chain, but specific to the PDP service (rather than generic Merkle tree ops that I moved to Proofs.sol).